client/my-sites/site-settings/controller.js

```diff
@@ -111,6 +112,11 @@ export function startSiteOwnerTransfer( context, next ) {
 	next();
 }
 
+export function headerCode( context, next ) {
+	context.primary = <HeaderCodeSettings />;
```

Comment:
Will all site types support this? I see Jetpack sites will, but what about WP.com simple / Atomic / etc.?

Reply:
The code in https://github.com/Automattic/jetpack/pull/34881 [explicitly excludes Simple](https://github.com/Automattic/jetpack/blob/287c3e786d56e29c0cea12d950a60ed1f63f9d29/projects/plugins/jetpack/modules/header-code.php#L20-L23). As things stand I don't see anything stopping it from Atomic.

Reply:
See my comments below!